enterprise/frontend/src/embedding-sdk/hooks/private/use-init-data.unit.spec.tsx

```diff
@@ -209,7 +209,10 @@ describe("useInitData hook", () => {
     });
 
     it("should use a custom fetchRefreshToken function when specified", async () => {
-      let fetchRequestToken = jest.fn(async () => ({ id: "foo", exp: 10 }));
+      let fetchRequestToken = jest.fn(async () => ({
+        id: "foo",
+        exp: Number.MAX_SAFE_INTEGER,
```

Comment:
Needed because we're now checking if `exp` is present and throwing an error if it's not

Reply:
It's a value that we now it won't make the test fail :)
We could calculate an expiration in the future with some math but this should be faster and more reliable (unless we start checking if it's a valid exp....)